splunk_add_on_ucc_framework/ucc_ui_lib/src/main/webapp/components/BaseFormView.jsx

```diff
@@ -191,15 +187,17 @@ class BaseFormView extends PureComponent {
                 this.setErrorMsg(error.errorMsg);
             }
         }
+        if (!error && this.hook && typeof this.hook.onSave === 'function') {
+            const validationPass = this.hook.onSave(datadict);
+            if (!validationPass) {
+                this.props.handleFormSubmit(false, false);
```

Comment:
nit: I think this method requires a little re-factoring to be readable. It is not clear what it is doing based on positional Booleans as argument.